api/v2/serializers/summaries/quota.py

```diff
@@ -20,3 +21,23 @@ class Meta:
             # networking
             'floating_ip_count', 'port_count',
             )
+
+
+class QuotaWritableSummarySerializer(serializers.RelatedField):
+    def get_queryset(self):
+        return Quota.objects.all()
+
+    def to_representation(self, model):
+        serializer = QuotaSummarySerializer(model, context=self.context)
+        return serializer.data
+
+    def to_internal_value(self, data):
+        try:
+            pk = data["id"]
+            return self.get_queryset().get(pk=pk)
+        except ObjectDoesNotExist:
+            raise serializers.ValidationError("Object does not exist for id: %s" % pk)
+        except (KeyError, TypeError, ValueError):
+            raise serializers.ValidationError(
+                "Expected dictionary with primary key like: { id: <Int> }. "
+                "Intead got: %s" % str(data))
```

Comment:
I believe this is pretty similar to `ModelRelatedField`? [Source](https://github.com/cyverse/atmosphere/blob/master/core/serializers/fields/base.py#L40-L46)